### DISCUSSION TITLE
How to jump over closing brackets?

### DISCUSSION THREAD

**Discussion by Dirleye** [zed-industries/zed#19200] [Feature Requests]
I'd like to be able to skip over closing quotation marks and brackets with the tab key, similarly to the "Tab Out" extension in VSCode or the default behaviour in JetBrains IDEs.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
